docs/cli/graphql-transformer/directives.md

```diff
@@ -886,7 +953,9 @@ type Post @model
 }
 ```
 
-In the example above the model is protected by Cognito User Pools by default and the `owner` can perform any operation on the `Post` type, but a Lambda function through the configured IAM policies can only call the ```getPost``` and ```listPosts``` query.
+> The above schema assumes a default authentication type of **Amazon Cognito User Pools** and a secondary authentication type of **IAM**.
+
+In the example above the `Post` model is protected by Cognito User Pools by default and the `owner` can perform any operation on the `Post` type. A Lambda function through the configured IAM policies can also call the `getPost` and `listPosts` query.
```

Comment:
"A Lambda function through the configured IAM policies?"

I'm not sure, but I think this means:

> You can also call `getPosts` and `listPosts` from an AWS Lambda function if it is configured with the appropriate IAM policies in its execution role.